plugins/bufbuild/connect-web/v0.2.1/Dockerfile

```diff
@@ -1,10 +1,11 @@
 # syntax=docker/dockerfile:1.4
-FROM node:16.17.1-bullseye-slim AS build
+FROM node:18.12.1-alpine3.16 AS build
```

Comment:
Why do we want to reduce the builder images?

Reply:
Not required but Node works well w/ alpine so figured I'd make the switch.

Reply:
I see, that does mean our Dockerfile's become slightly more complicated, but that's fine.

Wdyt about pinning to cached docker images on the Github runners? 

https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#cached-docker-images

Reply:
I think the cached image `node:18-alpine` will be used here - we're using a more specific tag to track version updates.